src/generate.js

```diff
@@ -142,10 +198,13 @@ const generate = async (tokens) => {
               }))
             ).json()
 
+            // Trigger review if code not verified on Etherscan
             if (result[0].ABI === "Contract source code not verified") {
-              console.log(`requires manual review: token code not verified on Etherscan on chain ${chain}`);
+              review(folder, chain, "code not verified on Etherscan");
             }
           }
+        } else {
+          review(folder, chain, "nonstandard token");
```

Comment:
nit: This is fine but I think it would be nicer if we instead stored the review triggers in an array and then at the very end

if (reviewReasons.length) {
  log(... reviewReasons)
} else {
  log('no review needed')

Reply:
sgtm, let's see how we feel about this. I might do a pass later to fix up stylistic stuff everywhere but should be fine for now.